### PR TITLE
WQP-1445

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/test/java/gov/acwi/wqp/etl/activity/AnalyzeActivityIT.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public void analyzeActivityFlowTest() {
 		try {
 			JobExecution jobExecution = jobLauncherTestUtils.launchJob(testJobParameters);
 			assertEquals(ExitStatus.COMPLETED, jobExecution.getExitStatus());
+			Thread.sleep(1000);
```
Comment (mbucknell) [inline src/test/java/gov/acwi/wqp/etl/activity/AnalyzeActivityIT.java]: Why is this needed?

File: src/test/java/gov/acwi/wqp/etl/summaries/organizationSum/TransformOrganizationSumIT.java [hunk 2]
```diff
@@ -50,19 +51,21 @@ public void setUp() {
...
-//	public void transformOrganizationSumStepTest() {
-//		try {
-//			JobExecution jobExecution = jobLauncherTestUtils.launchStep("transformOrganizationSumStep", testJobParameters);
-//			assertEquals(ExitStatus.COMPLETED, jobExecution.getExitStatus());
-//		} catch (Exception e) {
-//			e.printStackTrace();
-//			fail(e.getLocalizedMessage());
-//		}
-//	}
+//TODO - WQP-1406
```
Comment (mbucknell) [inline src/test/java/gov/acwi/wqp/etl/summaries/organizationSum/TransformOrganizationSumIT.java]: Remove?

File: src/test/java/gov/acwi/wqp/etl/activity/AnalyzeActivityIT.java (hunk 1 above)
Comment (dsteinich) [reply src/test/java/gov/acwi/wqp/etl/activity/AnalyzeActivityIT.java]: The analyze seems to be a little slow updating the catalog sometimes - tests sometimes fail without the sleep.

File: src/test/java/gov/acwi/wqp/etl/summaries/organizationSum/TransformOrganizationSumIT.java (hunk 2 above)
Comment (dsteinich) [reply src/test/java/gov/acwi/wqp/etl/summaries/organizationSum/TransformOrganizationSumIT.java]: 1406 is the json portion of the summary and still to be worked on. We need WQX data before that as it has worked in the past on the other ETLs, but not WQX...

Comment (dsteinich) [reply src/test/java/gov/acwi/wqp/etl/summaries/organizationSum/TransformOrganizationSumIT.java]: WQP-1406 is the JSON portion of the summary and blocked by lack of WQX data.
